### DISCUSSION TITLE
Changing surface with time

### DISCUSSION THREAD

**Discussion by Joe-Fitzy** [firemodels/fds#12841] [Q&A] [answered]
Hi all,

Currently I'm working on a supply/extract system and trying to change it to extract/extract at a certain time. 

When i have two vents overlapping the FDS comes back with an error. To try and alleviate this error i set the supply to only activate at 320 seconds and the other vent to deactivate at 315, but it still has the same error. The simulation still runs but it only activates the first vent in the coding. 

Does anyone know a way to change a surface after x amount of time?

Thanks a million,
Joe

**Comment by drjfloyd** [comment, marked as answer]
You cannot change the type of boundary condition applied to a solid surface. You can only start or stop any RAMPs (or TAUs) using device or control functions. 

If you are just looking to change the magnitude of a flow, temperature, or other variable controllable with a RAMP, then use a RAMP. 

If you are looking to change the type of boundary condition, create an OBST to cover the first VENT with the second VENT applied to it.

**Reply by Joe-Fitzy** [reply]
Thanks, the OBST is probably my best choice.
